decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb

```diff
@@ -3,6 +3,55 @@
     <h2 class="card-title"><%= t ".colors_title" %></h2>
   </div>
   <div class="card-section">
+
+    <details class="color-picker">
+      <summary>
+        <%= t(".title") %>
+      </summary>
+
+      <div class="color-picker__box">
+        <p><%= t(".explanation") %></p>
+
+        <label class="color-picker__label_choose">
+          <%= t(".choose_color") %>
+          <div id="primary-selector"></div>
+        </label>
+
+        <label class="color-picker__label_saturation" for="primary-saturation">
+          <%= t(".saturation") %>
+          <input id="primary-saturation" type="range" min="0" max="100">
+        </label>
+
+        <fieldset>
+          <legend><%== t(".legend", link: "https://www.color-meanings.com/triadic-colors/") %></legend>
```

Comment:
Since we are using erb escape, maybe we could use the `_html` convention. Also, since the URL is static, we could add it directly to the translation string. 
```suggestion
          <legend><%= t(".legend_html", link: "https://www.color-meanings.com/triadic-colors/") %></legend>
```

Reply:
Ok, applied both suggestions on 3b7cfe66e8cd66a45114447024b535d895970b9e. 
Just FYI, my reasoning was that as these links wouldn't be translated and they could actually introduce problems for translators, as there are too many characters that they don't need to change but they can mess up with. 
As this is not the conventional usage of translation variables in strings, then I prefer to do it as you propose 😄 
I've also changed it on the `colors_warning_html` key (see 8f23fda55234dd9748980b285d0d38e2b9cb9e90) so we're consistent on this. 
Leaving this as unresolved so you can ACK, review it, resolve it and finally merge it if you're happy.